proxy/common/flag.py

```diff
@@ -301,14 +307,7 @@ def initialize(
                 socket.AF_INET6 if args.hostname.version == 6 else socket.AF_INET
             )
         else:
-            # FIXME: Not true for tests, as this value will be a mock.
-            #
-            # It's a problem only on Windows.  Instead of a proper
-            # fix in the tests, simply commenting this line of assertion
-            # for now.
-            #
-            # assert args.unix_socket_path is None
-            args.family = socket.AF_INET6 if args.hostname.version == 6 else socket.AF_INET
```

Comment:
Wondering if this is intentional.  Can you provide some background.  Thank you

PS: I see all windows workflows are kind of broken now , e.g. https://github.com/abhinavsingh/proxy.py/actions/runs/4533948198/jobs/7987500924?pr=1325

Reply:
Looking into that issue right now as that code can't be valid with mixed IPv4/IPv6 addresses.

Reply:
Most tests mocking `socket.fromfd` are quite hard to redo w/o issues tbh

Reply:
> # assert args.unix_socket_path is None

The comment above this line indicates issue on windows after removing this line.  I'll need to get my hands dirty to recall more context around it.